Makefile

```
@@ -0,0 +1,5 @@
+build:
+	go build -o bin/ .
+
+fmt:
+	go fmt .
```

If you want to format the entire repo:

```suggestion
	go fmt ./...
```